.pre-commit-config.yaml

```
@@ -152,6 +152,11 @@ repos:
             \#\ type:\s?ignore(?!\[)
         language: pygrep
         types: [python]
+    -   id: np-bool
+        name: Check for use of np.bool instead of np.bool_
+        entry: np.bool[^_8]
```

this is matched using `re.search`, so perhaps it should be
```suggestion
        entry: np\.bool[^_8]
```
?

---

yeah, i tend to be lazy on those. I expect we have lots of cases in the pytest raises checks where we don't escape the period.